### PR TITLE
If the maintenance reason is a url, then redirect to this address

### PR DESCRIPTION
**Pull request by annando**
See the discussion here: https://github.com/friendica/friendica/issues/3242

### CODE REVIEW

File: util/maintenance.php [hunk 1]
```diff
@@ -49,5 +49,5 @@
 }
 
 echo "Usage:\n\n";
-echo "\tphp {$argv[0]} [1] [Maintenance reason]\tSet the system in maintenance mode\n";
+echo "\tphp {$argv[0]} [1] [Maintenance reason|redirection url]\tSet the system in maintenance mode\n";
```
Comment (tobiasd) [inline util/maintenance.php]: could you add an explanation what setting the reason or url results in?

Comment (annando) [inline util/maintenance.php]: I'm not really sure how to format this.

Comment (tobiasd) [inline util/maintenance.php]: Maybe below the examples as a test paragraph? One blank line and then the explanation text.

Comment (annando) [inline util/maintenance.php]: It now looks this way:
```Usage:

	php util/maintenance.php [1] [Maintenance reason|redirection url]
		Set the system in maintenance mode

		If the optionally entered maintenance reason is an url
		the visitor is redirected to that page.

		Examples:
			php util/maintenance.php 1 System upgrade
			php util/maintenance.php 1 http://domain.tld/downtime

	php util/maintenance.php 0
		Set the system in normal mode
```

**Pull request merged**